go/vt/sqlparser/sql.y

```diff
@@ -486,6 +497,16 @@ table_name_list:
     $$ = append($$, $3)
   }
 
+delete_table_list:
```

Comment:
Should this just include the `table_name` case as well, with the `DELETE comment_opt table_name_list` case above being dropped? It seems like this fails to parse `DELETE some_stuff.*, some_other_stuff FROM some_stuff, some_other_stuff;`